pkgs/applications/video/gpu-screen-recorder/default.nix

```diff
@@ -49,7 +51,7 @@ stdenv.mkDerivation {
     mv $out/bin/gpu-screen-recorder $out/bin/.wrapped/
     makeWrapper "$out/bin/.wrapped/gpu-screen-recorder" "$out/bin/gpu-screen-recorder" \
     --prefix LD_LIBRARY_PATH : ${libglvnd}/lib \
-    --prefix PATH : $out/bin
+    --suffix PATH : $out/bin
   '';
 
   meta = with lib; {
```

Comment:
Why? As far as I see most packages follow that syntax as a standard to avoid duplication of `lib.xxx`

Reply:
> It's apparently in bad form. I don't sway one way or the other, it's just most recent PRs I've seen/made they wanted this change done. https://github.com/NixOS/nixpkgs/pull/294383/files https://github.com/NixOS/rfcs/pull/120/files

Done.
But instead of `with lib.maintainers`, I used `lib.maintainers.xxx` as there is only one maintainer.